### PR TITLE
set-up-project-and-start-exercise-A (ticket 1 - issue id #1)

### PR DESCRIPTION
**Pull request by Sarahyoussef19**

### CODE REVIEW

File: Exercise-A-Static-page/user.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Exercise A</title>
+</head>
+
+<body>
+    <h1>Exercise A</h1>
+    Id: 1
+    Name: Sarah
+    Adress: Entença 95
```
Comment (francoiscrouzet) [inline Exercise-A-Static-page/user.html:15]: Could you put lastname and forename please?

**Review by francoiscrouzet — CHANGES_REQUESTED** [review]
Very good first PR @Zfitmeta ! 👍

File: Exercise-A-Static-page/user.html (hunk 1 above)
Comment (Zfitmeta) [reply Exercise-A-Static-page/user.html:16]: It's done. Thank You.

**Comment by Zfitmeta** [comment]
I also added break lines after each element.

File: Exercise-A-Static-page/user.html (hunk 1 above)
Comment (francoiscrouzet) [reply Exercise-A-Static-page/user.html:16]: Hey! Great
Last think I hope: can you keep the spacing around the colons consistent (sometimes blalba:blalba and sometimes blalba: blalba )
Also check what is the best practice in english, space before, after or both
thanks

**Review by francoiscrouzet — CHANGES_REQUESTED** [review]

File: Exercise-A-Static-page/user.html (hunk 1 above)
Comment (Zfitmeta) [reply Exercise-A-Static-page/user.html:16]: Done Thanks.

**Review by francoiscrouzet — APPROVED** [review]

**Comment by Sarahyoussef19** [comment]
https://github.com/Zfitmeta/Mentoring/issues/1

**Pull request merged**